src/Mono.Android/Mono.Android.targets

```diff
@@ -128,6 +129,7 @@
         SourceFiles="$(IntermediateOutputPath)__dex\classes.dex"
         DestinationFiles="$(OutputPath)mono.android.dex"
     />
+    <Touch Files="$(OutputPath)mono.android.dex" />
```

Comment:
I don't understand why you have to Touch this file, and I believe unnecessary touching is bad because it can cause possible later rebuild cascades, but anyhow changes in this code path will be simply lost because it will be killed because **it is jack**!
https://github.com/xamarin/xamarin-android/pull/514/files#diff-77c7d22cbe4c79b8f43dd2e97eb81688

It should have been actually already killed (I was totally unaware that jack could be used outside our MSBuild tasks).